test/mocks/v1/testing_snapshot.sk.go

```diff
@@ -257,6 +257,65 @@ func (s *TestingSnapshot) RemoveFromResourceList(resource resources.Resource) er
 	}
 }
 
+func (s *TestingSnapshot) RemoveAllResourcesInNamespace(namespace string) {
```

Comment:
Should we store resources indexed by namespace so that we can more easily find/delete them?